dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -98,9 +94,10 @@ printf "%23s: %s\n" "avr-libc" "$(avr_libc_version avr-gcc)"
 printf "\n"
 printf "%s\n" "Installed development tools"
 printf "%s\n" "---------------------------"
-for c in cmake cppcheck doxygen flake8 git make openocd python python2 python3; do
-    printf "%23s: %s\n" "$c" "$(get_cmd_version $c)"
+for c in cmake cppcheck doxygen git make openocd python python2 python3; do
+    printf "%23s: %s\n" "$c" "$(get_cmd_version "${c}")"
 done
+printf "%23s: %s\n" "flake8" "$(get_cmd_version "python3 -m flake8")"
```

Comment:
@smlng This should make the warning go away:

```suggestion
printf "%23s: %s\n" "flake8" "$(get_cmd_version "python3 -Wignore -m flake8")"
```